content/blogposts/2024/codyception-coding-cody-tools-with-cody.md

```diff
@@ -0,0 +1,159 @@
+      "context": {
+        "codebase": true
+      }
+    },
+    "evaluate-code-external-apis": {
+      "description": "evaluate-code-external-apis",
+      "prompt": "Evaluate and report on the usage and reliability of external APIs in the current project, suggesting updates or replacements.",
+      "context": {
+        "codebase": true
+      }
+    },
+    "implement-code-database-indexing": {
+      "description": "implement-code-database-indexing",
+      "prompt": "Implement code database indexing strategies in the current project, optimizing data retrieval and query performance.",
+      "context": {
+        "codebase": true
+      }
+```
+
+## The cody.json File: Tailoring Commands to Your Needs ##
+
+Customizing commands is made easy with the `cody.json` file. Here's how you can leverage it:
+
+### Project-Specific Commands:
+
+Create `cody.json` in the `.vscode` directory of your project to make commands accessible only in that workspace. Cody will execute these custom commands when working on the project.
+
+### Global commands:
+
+Create a new `cody.json` file in the `.vscode` directory of your home directory for commands accessible across multiple projects. Cody will be able to execute these global custom commands in any workspace.
+
+### Workspace vs. User Commands
+
+Understanding the scope of your custom commands is essential:
+
+- `cody.json` in a project's `.vscode` folder: Commands are available only in that workspace.
+- `cody.json` in the home `.vscode` folder: Commands are available globally across all workspaces.
+
+This flexibility allows you to organize project-specific commands separately from reusable commands accessible globally, enhancing your customization capabilities with Cody.
+
+## Cody's custom commands syntax
+
+### Running commands made simple
+Executing commands is a breeze with these options:
+
+- Type / in the chat box and select your desired command from the pop-up list.
+
+- Use ctrl+shift+v (cmd+shift+v on macOS) anywhere in your document to open the Command Menu, and just like magic, you can select and run commands without the chat sidebar.
+
+- Right-click on selected code to open the Cody context menu and choose a command.
+
+- Search for >Cody: Commands Menu in your Command Palette to access the Command Menu.
+
+## How I created the commands for [Codyception](https://cody.kynlo.co.uk) with the help of Cody ##
+
+
+Alright, buckle up for the extended saga of how I thrust Codyception into the coding cosmos. Picture this: innocently pondering the idea of Cody creating commands for me. It all started with a simple thought, "Hey, wouldn't it be cool?" But like any curious coder, I couldn't resist pushing the boundaries. "Hmm, is there a limit to these commands before something breaks?" I wondered. Little did I know, the quest for limits would propel me into a coding odyssey of epic proportions.
```

Comment:
I would remove or rework the rest of this to focus on the technology rather than the humor. I think the message really gets lost in the content.

Reply:
👍